lib/Parse/ParseDecl.cpp

```diff
@@ -829,12 +829,12 @@ ParserResult<DifferentiableAttr>
 Parser::parseDifferentiableAttribute(SourceLoc atLoc, SourceLoc loc) {
   StringRef AttrName = "differentiable";
   SourceLoc lParenLoc = loc, rParenLoc = loc;
-
+  
+  bool linear;
```

Comment:
Small update I wanted to do from last PR. Since `linear` is first argument in `@differentiable`, I also wanted it to be the first one declared